### ISSUE TITLE
Failed install

### ISSUE THREAD

**Issue by mroberts1** [makeworld-the-better-one/gemget#17]
>sudo make install generates the message below on MacOS 10.15 - any idea why?

--------
install -Dm 755 gemget /usr/local/bin/gemget
install: illegal option -- D
usage: install [-bCcpSsv] [-B suffix] [-f flags] [-g group] [-m mode]
               [-o owner] file1 file2
       install [-bCcpSsv] [-B suffix] [-f flags] [-g group] [-m mode]
               [-o owner] file1 ... fileN directory
       install -d [-v] [-g group] [-m mode] [-o owner] directory ...
make: *** [install] Error 64

**Comment by makew0rld** [comment]
Fixed in 60dec7c, thanks for reporting! It appears macOS has a different version of the install command.

**Comment by makew0rld** [comment]
This needs to be changed to use `-d`.

On Linux, with GNU coreutils for `install`:

```
       install [OPTION]... -d DIRECTORY...

...

       -d, --directory
              treat all arguments as directory names; create all components of the specified directories 

       -D     create all leading components of DEST except the last, or all components of --target-directory, then copy SOURCE to DEST 
```

On macOS with BSD version of `install`:

```
     -d      Create directories.  Missing parent directories are created as required. 

(No -D option)
```

The solution is to use `-d` and create the directory, and then install the binary.

This also applies to Amfora's Makefile.

**Comment by makew0rld** [comment]
The issue described in the above comment was fixed in 82be646.

It was fixed in https://github.com/makeworld-the-better-one/amfora/commit/f9327c0a5105b42a6920458015553454b21eb92b for Amfora.
